shared/common-adapters/with-tooltip.desktop.js

```diff
@@ -53,7 +55,7 @@ class WithTooltip extends React.Component<Props, State> {
               this.props.multiline && styles.containerMultiline,
             ])}
             visible={!!this.props.text && this.state.visible}
-            attachTo={() => this._attachmentRef}
+            attachTo={this.state.attachmentRef && this._getAttachmentRef}
```

Comment:
I don't think `this.state.attachmentRef &&` and the `attachTo` type change is necessary, the behavior will be the same if `this._getAttachmentRef` returns null.

Reply:
It's necessary because if you just use `this._getAttachmentRef` the prop `attachTo` doesn't change whether `this.state.attachmentRef` is null or not. This was the same bug that affected https://github.com/keybase/client/pull/15300.